### ISSUE TITLE
[REQUEST] Exclude deprecated mods from search results

### ISSUE THREAD

**Issue by DRY411S** [arielbeje/uBot#21]
I'm finding the code too hard to understand to produce a pull request.

But I'm guessing that some tests around line 353 in factorio.py commit 0725a8b would do the trick.

**Comment by SuperSandro2000** [comment]
If the mod portal shows those mods the bot should show it, too.

**Comment by DRY411S** [comment]
That would be fair enough if the bot wasn't restricted to 5 results, unlike the portal.

**Comment by SuperSandro2000** [comment]
Yeah... Maybe we should show more results or a link to the mod portal? But not finding archived mods makes the bot weird if you are searching for archived mods.

**Comment by DRY411S** [comment]
I assumed that the restriction to 5 results was due to the run time of the bot and all the screenscraping/beautiful soup parsing.

Maybe, the bot output could include a 'more results at' link, with the URL that the bot has just used?

My request isn't about wanting to hide deprecated results, it's more about wanting to display more useful results.

With that in mind, maybe "?version=any" in the screencrape URL should be "?version=1.0" now the game is released?

**Comment by SuperSandro2000** [comment]
> With that in mind, maybe "?version=any" in the screencrape URL should be "?version=1.0" now the game is released?

any means any version cause if you want to find a mod which is not released for the current version you can't find it.

**Comment by DRY411S** [comment]
Sure. But is also means 'all' including unsupported versions.
